cypress/support/commands.js

```diff
@@ -24,47 +24,56 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
-const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
+// Логін користувача через API
+// Реєстрація користувача через API
+// Реєстрація користувача через API
 
+Cypress.Commands.add('registerUser', (email, username, password) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users',
+    body: {
+      user: {
+        email,
+        username,
+        password
+      }
+    },
+    failOnStatusCode: false // Додано, щоб уникнути помилки через вже зайнятий email
+  }).then((response) => {
+    if (response.status === 422) {
+      cy.log('Ця електронна пошта або ім\'я користувача вже використовується.');
+    } else {
+      expect(response.status).to.eq(200); // Перевірка, що користувач зареєстрований
+    }
+  });
+});
+
+// Логін користувача через API
 Cypress.Commands.add('login', (email, username, password) => {
```

Comment:
Great use of custom commands for user login. This will help you to avoid repetitive code in your tests.